src/styles/card/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: $card-width;
+  background-color: $color-white;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px ;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__product-name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: $color-main-accent;
+  }
+  
+  &__product-code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-secondary;
+    margin-bottom: 16px;
+  }
+  
+  &__product-rate {
+    display: flex;
+    margin-bottom: 24px;
+    justify-content: space-between;
+  }
+  
+  &__product-review {
+    display: inline-block;
+    font-size: 10px;
+    color: $color-main-accent;
+    margin-left: 18px;
+    padding-top: 2px;
+  }
+  
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+  
+  &__price-word {
+    color: $color-secondary;
+    font-size: 12px;
+    padding-top: 2px;
+  }
+  
+  &__price-number {
+    color: $color-main-accent;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+  
+  &__button-buy {
+    box-sizing: border-box;
+    display: inline-block;
+    background-color: $color-blue;
+    border-radius: 5px;
+    outline: none;
+    text-decoration: none;
+    text-align: center;
+    color: $color-white;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+    padding: 12px 70px;
+  }
+  
+  &__button-buy:hover {
+    background-color: $color-white;
+    color: $color-blue;
+    border: 1px solid $color-blue;

```

Comment:
move this to `&__button-buy` so block wouldn't change its width on hover

Reply:
done/ thanks for advise